src/ui/virtualkeyboard/virtualkeyboard.cpp

```diff
@@ -120,6 +125,16 @@ class VirtualKeyboardBackend
     VirtualKeyboard *parent_;
 };
 
+void VirtualKeyboardBackend::setVirtualKeyboardFunctionMode(uint32_t mode) {
+    if (mode != static_cast<uint32_t>(VirtualKeyboardFunctionMode::Full) &&
+        mode != static_cast<uint32_t>(VirtualKeyboardFunctionMode::Limited)) {
+        return;
```

Comment:
throw invalid argument error here, 

See:
https://github.com/fcitx/fcitx5/blob/00d52a4a34aaabd2fac922deff1860c5ef04e759/src/modules/dbus/dbusmodule.cpp#L295

Reply:
fix it.

Reply:
> throw invalid argument error here,
> 
> See:
> 
> https://github.com/fcitx/fcitx5/blob/00d52a4a34aaabd2fac922deff1860c5ef04e759/src/modules/dbus/dbusmodule.cpp#L295

fix it.